scripts/pdf/render.py

```diff
@@ -15,7 +15,8 @@
 import argparse
 from datetime import datetime
 
-from weasyprint import HTML, CSS
+from weasyprint import HTML
+
```

Comment:
Why the extra empty line?

Reply:
Per pep8, there should be two empty lines before any function (and class) definition.

Functions within a class only need one empty like before the function definition (except the first function).